src/main/java/org/apache/commons/lang3/ClassUtils.java

```diff
@@ -1059,6 +1067,62 @@ public static boolean isInnerClass(final Class<?> cls) {
         return getClass(loader, className, initialize);
     }
 
+    /**
+     * Returns a list of base classes/interfaces underneath the supplied package
```

Comment:
I believe it would be better with dots in the end of sentence.

Reply:
I have committed changes for this